rtsp-to-web/config.yaml

```diff
@@ -21,6 +21,10 @@ options:
   http_port: 8083
   rtsp_port: 5541
   insecure_skip_verify: false
+  ice_servers:
+    - stun:stun.l.google.com:19302
+  ice_username: ""
```

Comment:
Do you think many people will actually run their own server and this? I'm curious if we just shouldn't bother in the name of simplicity and let people that want to run their own stun server that has credentials just setup a standalone instance of this service.

Or is this actually a popular thing that people have? e.g. provided by their cloud provider or something.

Reply:
You're probably right. Perhaps we just make it a boolean that people toggle (i.e "Use Google STUN servers for NAT Traversal") - would that work for you?

Reply:
I'd say there are so many network configs out there that people might need their own STUN/TURN servers. Especially if they want to start using this with external access for things like Alexa/Google Home which would be an obvious Home Assistant use case. If we have good defaults I don't see why having the extra options hurts. @allenporter I'd prefer to use your addon than roll my own.

This morning I was trying to investigate if the random failures I get when using this as part of an Alexa doorbell are related to STUN/TURN (https://github.com/deepch/RTSPtoWeb/issues/132) so I'd be interested in these being configurable

Reply:
or at the very least move the config.json to the /data area so that advanced users can work with it?

Reply:
OK, lets keep the configuration options here, i'm sold.

w.r.t. /data. This is stored in /data https://github.com/allenporter/stream-addons/blob/3de0e3f19c14fb416955398482694523802c7356/rtsp-to-web/rootfs/etc/cont-init.d/01-rtsp-to-web#L9 

Is there a doc somewhere that explains how users can manually edit configuration files in /data?

Reply:
I think the ssh add-on and samba add-on are often mentioned but I've always run on my own OS with SSH access anyway so I've never looked into them.